README.md

```diff
@@ -22,7 +22,6 @@ Mark Thanadabouth, Software Developer
 Ayrat Gimranov, Software Developer
 Tom McGuire, Software Developer
 
-
 ## References
 
 [UML](/Users/veebs/Documents/CodeFellows/401/basic-express-server/src/UML.png)
```

Comment:
This should be just be`[UML](../src/UML.png) or move the photo to the root(same place as the readme.